src/test/resources/features/Scoring/Student_Quiz/StudentQuiz.feature

```diff
@@ -0,0 +1,29 @@
+Feature: Student Quiz
+
+  Background:
+    Given user prepare auth request
+    When user do login with email "admin@admin.com" and password "administratorfunctionapp"
+    And user prepare quiz request with batchCode "futur3"
+    And user hit get all quiz endpoint
+    And user get first quiz and store id
+    And user prepare student quiz request with stored quiz id and batchCode "futur3"
```

Comment:
Also, does this background require batch with batchCode `futur3` to exist prior to running this feature's tests?